pkg/backend/httpstate/client/client.go

```diff
@@ -205,7 +205,7 @@ var (
 // GetLatestConfiguration returns the configuration for the latest deployment of a given stack.
 func (pc *Client) GetLatestConfiguration(ctx context.Context, stackID StackIdentifier) (config.Map, error) {
 	latest := struct {
-		Info apitype.UpdateInfo `json:"info,allowEmpty"`
+		Info apitype.UpdateInfo `json:"info,omitempty"`
```

Comment:
Would appreciate a second look here as my confidence in this particular change isn't high. `allowEmpty` looks to be a typo as I can't find any references to it anywhere else in the codebase or in golang docs online.

Reply:
`omitempty` looks good 👍